### ISSUE TITLE
Inconsistent spacing

### ISSUE THREAD

**Issue by flibbertigibbet** [azavea/climate-change-lab#13]
Some files use two spaces, some four. Should use one or the other for all files.

**Comment by CloudNiner** [comment]
Definitely. Angular 2 style is to use two space indentation. We could use an http://editorconfig.org/ file to make this explicit in the project. There are plugins for all the editors.
